Block/TagCloudBlock.php

```diff
@@ -17,16 +16,15 @@
 use Sonata\AdminBundle\Form\FormMapper;
 use Sonata\AdminBundle\Validator\ErrorElement;
 use Sonata\BlockBundle\Block\BaseBlockService;
-use Doctrine\ODM\PHPCR\DocumentManager;
 use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
 use Symfony\Cmf\Bundle\BlogBundle\Repository\TagRepository;
 
 /**
- * Tag Cloud block service
+ * Tag Cloud block service.
  *
  * @author Daniel Leech <daniel@dantleech.com>
  */
-class TagCloudBlockService extends BaseBlockService
+class TagCloudBlock extends BaseBlockService
```

Comment:
I think this entire class is pretty broken as ``TagRepository`` also doesn't exist .. it also appears like the class isn't even configured as a service either.